### PR TITLE
[data]: [manthanank]

### PR DESCRIPTION
**Pull request by manthanank**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3396"></a>

### CODE REVIEW

File: data/manthanank.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        "icon": "FaLink"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/manthanank/",
+        "icon": "FaLinkedin"
+      }
+    ],
+    "milestones": [],
+    "testimonials": []
```
Comment (Panquesito7) [inline data/manthanank.json:31]: You can remove these fields if you're not going to use them for now.
```suggestion
```

File: data/manthanank.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  }
+
```
Comment (Panquesito7) [inline data/manthanank.json:33]: There's an extra line here.
```suggestion
}
```

File: data/manthanank.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+    "name": "Manthan Ank",
+    "type": "personal",
+    "displayStatsPublic": true,
+    "bio": "**Front End** Developer | Angular | JavaScript | Nodejs | MongoDB | Open Source",
+    "avatar": "https://github.com/manthaank.png",
```
Comment (Panquesito7) [inline data/manthanank.json:6]: Wrong username.
```suggestion
    "avatar": "https://github.com/manthanank.png",
```

File: data/manthanank.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    "tags": ["Open Source", "Javascript", "Typescript", "Angular", "Nodejs", "MongoDB"],
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/manthaank",
```
Comment (Panquesito7) [inline data/manthanank.json:11]: Same here.
```suggestion
        "url": "https://github.com/manthanank",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by manthanank** [comment]
updated

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good, Great Collaboration Everyone! 🎉

**Comment by manthanank** [comment]
Finally, on LinkFree after so many conflicts✌️😂, Thanks

**Pull request merged**